app/build.gradle

```diff
@@ -1,37 +1,21 @@
 apply plugin: 'com.android.application'
-import com.android.builder.core.DefaultManifestParser
 
 android {
-    signingConfigs {
-        signingkey {
-            storeFile file('/home/dave/.android-keystore/playstorekeys')
-        }
-    }
-    compileSdkVersion 19
-    buildToolsVersion "21.1.2"
+    compileSdkVersion 29
+    buildToolsVersion "29.0.3"
     defaultConfig {
         applicationId "net.justdave.mcstatus"
         minSdkVersion 19
-        targetSdkVersion 19
+        targetSdkVersion 29
     }
     buildTypes {
         release {
             minifyEnabled false
             proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.txt'
-            signingConfig getSigningConfig()
-            applicationVariants.all { variant ->
-                variant.outputs.each { output ->
-                    def file = output.outputFile
-                    def manifestParser = new DefaultManifestParser()
-                    def String versionName = manifestParser.getVersionName((File) android.sourceSets.main.manifest.srcFile)
-                    def fileName = project.getRootProject().name + "-" + versionName + ".apk"
-                    output.outputFile = new File(file.parent, fileName)
```

Comment:
Removing this makes the filename be "app-debug.apk" or "app-release.apk" instead of "mcstatus-1.0.3.apk". Is there a better way to do this?

Reply:
yeah, I haven't built this in a while :-(  this code doesn't even work anymore which is probably why you removed it.  I have other projects that do it a different way that still works, so I'll fix that after I merge this.